.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "8"
+
+install:
+  - npm install
+
+script:
+  - npm test
```

Comment:
both `npm install` and `npm test` are the defaults for either configuration so we can leave them out. But we can also make it explicit if you prefer